webpack/assets/javascripts/react_app/components/common/table/components/NameCell.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { urlBuilder } from '../../../../common/urlHelpers'
 
 const NameCell = ({ active, id, name, controller, children }) =>
   active ? (
-    <a href={`/${controller}/${id}-${name}/edit`}>{children}</a>
+    <a href={urlBuilder(controller, 'edit', id)}>{children}</a>
```

Comment:
Some pages use `${id}-${name}` and I don't know if leaving out the name works.

Reply:
now it's not used elsewhere than models and there it works, I believe both should work, as it's how Rails works, but I we could have some custom logic somewhere...

Reply:
but no biggie, let's not change it for now :)